wiki/Article_Styling_Criteria/en.md

```diff
@@ -1,581 +1,559 @@
-
-### Serial comma
-
-_The serial comma is also known as the "Oxford comma" or the "Harvard comma"._
-
-When a list consists of 3 or more items in a sentence, use the serial comma. For example:
-
-- The game modes of _osu!_: osu!standard, osu!taiko, osu!catch`,` and osu!mania are fun to play with others.
-
-Keep in mind that there are rare expections to not using the serial commas.
-
-### Game modes
-
-When writting the name of the game modes, they are to be written as follows:
-
-- `osu!`
-  - `osu!standard` (unofficial, but used to prevent ambiguity)
-    - for folder names, it is required to use `osu!`, not `osu!standard`
-- `osu!taiko`
-- `osu!catch`
-- `osu!mania`
-
-You may use the old game mode names (e.g. `Catch the Beat`, `Taiko`, or `Mania`) only when discussing about said game mode's previous name.
-
-### Game modifiers
-
-Game modifiers **must** be capitalised. For example:
-
-- `Hard Rock`
-- `Nightcore`
-- `Double Time`
-- `Easy`
-
-### Gameplay elements
-
-Gameplay elements should **never** be capitalised. For example:
-
-- In osu!standard, **beatmaps** are composed of three different gameplay elements: **circles**, **sliders**, and **spinners**.
-- The **beatmap** **editor** is a place where **beatmappers** can map a song of their choice.
-
-### Contractions
-
-Contractions are a shortened form of a word or group of words.
-
-In the osu!wiki, contractions are **heavily** discouraged. This is to keep a formal tone in the osu!wiki.
-
-For example, use:
-
-- `Do not` instead of `Don't`
-- `You will` instead of `You'll`
-- `It is` instead of `It's`
-
-Note: the possessive forms of words (e.g. `the player's` or `the skinner's`) are not contractions.
-
-### Point of view
-
-When referring to the player, use `the player` or `a player`.
-
-**Never** use `he`/`his` or `she`/`her`! Instead, use `they`/`their`/`theirs`.
-
-In instances where you are referring directly to the reader (e.g. tutorial page), you could use the word `you`.
-
-**Never** use the first person perspective, `I`.
-
-### Terminology
-
-These words are spelt as follows (note the space):
-
-- `hit circles`
-- `approach circles`
-- `slider ticks`
-- `slider ball`
-- `slider path`
-- `hit burst`
-- `game mode` (or just `mode`)
-- `game mod` (or just `mod`)
-- `play style`
-- `hit sound`
-
-The term `Difficulty` refers to a specific `Beatmap` (these terms are interchangable). The term `Beatmap` may refer to a `Beatmapset`; however, to prevent ambiguity, a beatmapset is the collection of beatmaps per song per creator while a beatmap is the playable difficulty.
-
----
-
-Use "beatmaps" instead of "maps".
-
-Use "creator" instead of "beatmapper" or "mapper".
+# Article Styling Criteria
+
+_See also: [Article Styling Critria/News](/wiki/Article_Styling_Criteria/News)._
+
+The article styling crtieria serves as a enforced styling standard to keep consistency in clarity, formatting, and layout in all articles of the osu!wiki.

```

Comment:
`The article styling crtieria serves ...` ->
`The article styling criteria (ASC) serves ...`
- add `(ASC)`
- typo `crtieria` -> `criteria`